services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusControllerITest.kt

```diff
@@ -246,6 +249,85 @@ class HakemusControllerITest(@Autowired override val mockMvc: MockMvc) : Control
         }
     }
 
+    @Nested
+    inner class Create {
+        private val url = "/hakemukset"
+        private val hankeTunnus = "HAI24-94"
+
+        val request = CreateHakemusRequestFactory.johtoselvitysRequest(hankeTunnus = hankeTunnus)
```

Comment:
Should we test Kaivuilmoitus requests as well?

Reply:
I added a test for kaivuilmoitus.